core/src/test/java/org/kin/sendkin/core/UtilsUnitTest.java

```diff
@@ -0,0 +1,65 @@
+package org.kin.sendkin.core;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.kin.sendkin.core.model.RecipientContact;
+import org.kin.sendkin.core.view.Utils;
+
+import java.util.ArrayList;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+
+public class UtilsUnitTest {
+
+    private final String address1 = "GCCLBNMWRUKPCWV4J6H42KAQQ6GSOG6YKRNONAT3QDOGRH3T3EQGSXJR";
+    private final String name1 = "James Bond";
+
+    private final String address2 = "GDK57YFZRKNKJTRLE3CZ2GNQCSRC5NPJQWIEMD43M7V6ZIHNARGBIMFI";
+    private final String name2 = "Mr Bond";
+
+    private RecipientContact contact1;
+    private RecipientContact contact2;
+
+
+    @Before
+    public void initMocks() {
+        contact1 = new RecipientContact(name1, address1);
+        contact2 = new RecipientContact(name2, address2);
+    }
+
+
+    @Test
+    public void parseTest() {
```

Comment:
check the sad path of errors (see the comment of catching parsing errors, can be done in a different task)